molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/urls.py

```diff
@@ -22,6 +23,8 @@
     urlpatterns = patterns('', )
 
 urlpatterns += patterns(
+    '',
+    url(r'^smart-css/$', SmartCSS.as_view(), name='smart-css'),
```

Comment:
You can simplify this by using `TemplateView.as_view(template_name='smart-css.html')`